src/Wpf.Ui.Gallery/Views/Pages/Media/ImagePage.xaml

```diff
@@ -41,10 +41,13 @@
                     <controls:GalleryControlPresenter
                         Grid.Row="0"
                         Margin="0"
-                        CodeText="&lt;Image Height=&quot;100&quot; Source=&quot;Assets\MyImage.jpg&quot; /&gt;"
-                        HeaderText="A basic Image from a local file">
```

Comment:
I'd recommend leaving the default `Image` and adding a second one below, `ui:Image`, to show that there is a choice.